doc/source/whatsnew/v0.23.0.txt

```diff
@@ -523,6 +523,37 @@ Notice in the example above that the converted ``Categorical`` has retained ``or
 
 Note that the unintenional conversion of ``ordered`` discussed above did not arise in previous versions due to separate bugs that prevented ``astype`` from doing any type of category to category conversion (:issue:`10696`, :issue:`18593`).  These bugs have been fixed in this release, and motivated changing the default value of ``ordered``.
 
++.. _whatsnew_0220.api_breaking.pretty_printing:
+
+Better pretty-printing of DataFrames in a terminal
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Previously, the default values for the maximum number of rows and columns were
+``pd.options.display.max_rows=60`` and ``pd.options.display.max_columns=120``,
+respectively. This meant that relatively large frames would not fit within the
+terminal width or height, and Pandas would introduce line breaks to display
+these 20 columns. This resulted in an output that was relatively difficult to
+read:
+
+.. image:: print_df_old.png
+
+Starting with Pandas 0.22, the default value of ``pd.options.display.max_rows``
```

Comment:
you can just say the default now is 20 (don't need to say the version)

Reply:
Done